driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -113,6 +119,11 @@ public TrustStrategy trustStrategy()
         return trustStrategy;
     }
 
+    public AuthToken authToken()
+    {
+        return authToken;
+    }
+
```

Comment:
Messing with the heap can be restricted with a security policy but so can the set of callable methods so I guess that's a fair point.